packages/components/ContextualMenu/src/ContextualMenuItem.types.ts

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react';
+import { ViewProps, ImageProps } from 'react-native';
+import { IRenderData } from '@uifabricshared/foundation-composable';
+import { ITextProps } from '@fluentui-react-native/text';
+import { IPressableProps } from '@fluentui-react-native/pressable';
+import { ITextTokens, IForegroundColorTokens, IBackgroundColorTokens, IBorderTokens } from '@fluentui-react-native/tokens';
+import { IFocusable, IPressableState } from '@fluentui-react-native/interactive-hooks';
+import { IViewWin32Props } from '@office-iss/react-native-win32';
+
+export const contextualMenuItemName = 'ContextualMenuItem';
+
+export interface IContextualMenuItemState extends IPressableState {
+  /**
+   * Whether the menu item is clicked
+   */
+  // clicked?: boolean;
```

Comment:
Same question as above, not sure about leaving the commented code. Maybe someone else can chime in?

Reply:
I would generally remove it before checkin unless there is value in leaving it in. An example might be comments that are indicative of a future direction or a proposal. In that case I might add some context around why the commented code is present. Otherwise I would remove it.

Reply:
I'll remove them. Thanks!